Mission 3/NMCA_003/NMCA_003_strings.lua

```diff
@@ -1,9 +1,455 @@
+
+            Unlocked Tech:
+                T2 Static Shield - 3 minutes into the mission
+
+        Mission 2:
+            Intro:
+                Cinematics start at player's ACU, "scanning area aroud for more crystals". Player is assigned to move to the west (that's where the attack came from) to reclaim 3 more crystals that are located there,
+                and to deal with the Aeon bases before they are significant threat.
+
+            Primary:
+                Reclaim 3 more crystals to repair more ship's systems.
+
+                After some minutes, cinematics to the off map at gating Aeon ACU.
+                When player reclaims the crystals or several minutes later, Aeon counter attack coming from the north. This will expand the map to the north/main Aeon base.
+
+            Secondary:
+                Destroy 2 Aeon bases. This is no longer a primary objective.
+
+            Bonus:
+                Destroy over X Submarine Hunters (exact number in the script)
+                Shoot down engineer drop from south base (this will cause one less enemy base in the final part)
+
+            Unlocked Tech:
+                Railgun boat - 5 minutes into the mission
+                T2 Arty - 8 min into the mission
+
+        Mission 3:
+            Intro:
+                Cinematics at the incoming attack from the main base (north-west).
+
+                Frist orbital canon will be on the north island.
+
+            Primary:
+                Defend the Aeon counter attack.
+
+                After few minutes or once player scouts the orbital canon the map will expand to the east (last part).
+
+            Secondary:
+                Defeat the enemy ACU.
+
+            Unlocked Tech:
+                TMD/TML - once player scouts enemy TML (on the north island)
+
+        Mission 4:
+            Primary:
+                Destroy all 4 Orbital canons.
+        
+        Outro:
+            Cinematics at the ship leaving the planet.
+--]]
+
+
+
+-------------
 -- Debriefing
+-------------
+-- Showed on the score screen if player wins
 Debriefing_Win = {
     {text = 'You win', faction = 'UEF'},
 }
 
+-- Showed on the score screen if player loses
 Debriefing_Lose = {
     {text = 'You lose', faction = 'UEF'},
 }
--- Main Story
\ No newline at end of file
+
+
+
+-------------
+-- Win / Lose
+-------------
+-- When the ship dies
+ShipDestroyed = {
+    {text = '[Nichols]: The ship has been killed, mission failed.', vid = 'N03_ShipDestroyed.sfd', bank = 'N03_VO', cue = 'ShipDestroyed', faction = 'UEF'},
+}
+
+-- Player dies
+PlayerDies = {
+    {text = '[Nichols]: Player dead.', vid = 'N03_PlayerDies.sfd', bank = 'N03_VO', cue = 'PlayerDies', faction = 'UEF'},
+}
+
+
+
+------------
+-- Mission 1
+------------
+-- Look at the Aeon base
+M1Intro1 = {
+    {text = '[Benson]: We\'re picking up the ship\'s beacon somewhere from this position.', vid = 'N03_M1Intro1.sfd', bank = 'N03_VO', cue = 'M1Intro1', faction = 'UEF'},
```

Comment:
indeed. Thats what I was thinking as well. At the start of the mission we shouldnt be able to tell that the enemies are aeon. A good moment to find out would be capturing the civilian building. Since that's optional, we also need to have a second trigger.

That could be:
- Tech guys unscramble the communication
- last expansion of the map (there is an acu which could contact you), this would mean that we have to add a lot of dialogs twice.